bot/seasons/easter/april_fools_vids.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+
+class AprilFoolVideos(commands.Cog):
+    """A cog for april fools that gets a random april fools video from youtube."""
+    def __init__(self, bot):
+        self.bot = bot
+        self.yt_vids = self.load_json()
+        self.youtubers = ['google']  # will add more in future
+
+    @staticmethod
+    def load_json():
+        """A function to load json data."""
+        p = Path('bot', 'resources', 'easter', 'april_fools_vids.json')
+        with p.open() as json_file:
+            all_vids = load(json_file)
+        return all_vids
+
+    @commands.command(name='fool')
+    async def aprial_fools(self, ctx):
+        """Gets a random april fools video from youtube."""
+        random_youtuber = random.choice(self.youtubers)
+        category = self.yt_vids[random_youtuber]
+        random_vid = random.choice(category)
+        embed = discord.Embed()
```

Comment:
These 5 embed generation lines can be removed now that you're utilizing Discord's native handling for youtube links.